storage/innobase/handler/handler0alter.cc

```diff
@@ -1504,7 +1501,9 @@ instant_alter_column_possible(
 		= ALTER_ADD_STORED_BASE_COLUMN
 		| ALTER_DROP_STORED_COLUMN
 		| ALTER_STORED_COLUMN_ORDER
-		| ALTER_COLUMN_NULLABLE;
+		| ALTER_COLUMN_NULLABLE
+		| ALTER_COLUMN_UNVERSIONED
+		| ALTER_COLUMN_EQUAL_PACK_LENGTH;
```

Comment:
The intention of this collection of flags was to enumerate all operations where we can play some tricks to avoid rebuilding a table, by introducing more overhead to the DML code path. Neither `ALTER_COLUMN_UNVERSIONED` nor `ALTER_COLUMN_EQUAL_PACK_LENGTH` fall in this category; they were always part of `INNOBASE_ALTER_INSTANT` (which admittedly has a misleading name; maybe `INNOBASE_ALTER_METADATA` could be better).